modules/_labs/header/react/README.md

```diff
@@ -27,7 +27,6 @@ This component renders a responsive, Canvas-style header.
 ```tsx
 import * as React from 'react';
 import {Header} from '@workday/canvas-kit-labs-react-header';
-import {AvatarButton} from '@workday/canvas-kit-react-avatar';
```

Comment:
This line shouldn't have been deleted - should have just been changed to `Avatar`